constants.py

```diff
@@ -113,6 +113,7 @@ def _merge_vars(base_vars: JsonType, vars: JsonType) -> None:
 # Misc
 DEFAULT_LANG = "English"
 BASE_URL = URL("https://www.twitch.tv")
```

Comment:
Assuming the `channel.py` and `twitch.py` imports being removed, this could actually be dropped as a constant entirely, by incorporating this line into `ANDROID_APP` and `WEB` `ClientInfo`s.